bin/infra-deploy-status-check-configs

```diff
@@ -47,6 +47,8 @@
 # -----------------------------------------------------------------------------
 set -euo pipefail
 
+source bin/util.bash
```

Comment:
This one ends in `.bash` but the other ones don't?

Reply:
following google's shell style guide https://google.github.io/styleguide/shellguide.html#file-extensions

> Executables should have no extension (strongly preferred) or a .sh extension. Libraries must have a .sh extension and should not be executable.
>
> It is not necessary to know what language a program is written in when executing it and shell doesn’t require an extension so we prefer not to use one for executables.
>
> However, for libraries it’s important to know what language it is and sometimes there’s a need to have similar libraries in different languages. This allows library files with identical purposes but different languages to be identically named except for the language-specific suffix.

Reply:
nit, though following those guidelines it should be `bin/util.sh` not `bin/util.bash` right?

> Libraries must have a .sh extension